src/main/java/mate/academy/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,24 @@
 package mate.academy.dto;
 
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotBlank
+    @Size(min = 3, max = 64)
     private String title;
+    @NotBlank
+    @Size(min = 3, max = 64)
     private String author;
+    @NotBlank
+    @Size(min = 10, max = 64)
     private String isbn;
+    @NotBlank
```

Comment:
`@NotBlank` is used for String
```suggestion
    @NotNull
```